staging/kos/Makefile

```diff
@@ -2,26 +2,36 @@ DOCKER_PREFIX=${LOGNAME}
 
 publish:: publish-network-apiserver
 publish:: publish-ipam-controller
+publish:: publish-ca-controller
```

Comment:
"CA" stands for "Connection Agent", and including both "agent" and "controller" in this thing's name is redundant.  Call it the connection agent or the connection controller.

Reply:
I agree that having both "agent" and "controller" is redundant, but that redundance is intentional. It's an agent because it's a node-local "agent" of the SDN control plane. Also, it's a K8s controller because it talks to the API server and drives the actual state towards the desired one, hence the word "controller". "Agent" refers to the specific logic of the application, "controller" to a design pattern it implements. Nevertheless I am not religious about this and if your still against this redundancy I am ready to change the name. Another issue is that naming of the connection agent files and entities in the cluster (yaml manifests, service accounts, etc...) is inconsistent, in some of them it's just "ca", in other "ca-controller", I was planning to fix this soon.

Reply:
In a kube-style control plane, agents are generally controllers.  So I think it's redundant to say both "agent" and "controller" in the name.  "Connection agent" is fine since "agent" implies "controller".  "Connection controller" is also fine since the subject matter of connections is node-specific.

Reply:
Renamed connection agent related entities from ca-controller to connection-agent in 2598689.
Naming is still inconsistent in the source files. More specifically consider pkg/controllers/connectionagent/connectionagent.go pkg/controllers/ipam/ipam.go. For the ipam the name of the source file carries no information on the fact that it is a controller, whereas in connectionagent.go the name reveals that is is an agent. I think it's ok not to convey that information in the names of the source files as it is implied by the fact that they are under dir pkg/**controllers**, but then I would not know how to name the connectionagent source file ("connection" seems to me not enough). In any case IMHO this is not a priority now.